revent/settings.py

```diff
@@ -130,3 +130,5 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+DJANGO_SETTINGS_MODULE = "testing.settings"
```

Comment:
does `testing.settings` exist?
why are we pointing django to the testing settings?

Reply:
Removed